src/org/zaproxy/zap/view/BackgroundImagePanel.java

```diff
@@ -75,7 +87,9 @@ public final void setBackgroundImage(URL imageUrl) {
     protected void paintComponent(Graphics g) {
         if (img != null) {
             setOpaque(false);
-            g.drawImage(img, 0, 0, DisplayUtils.getScaledSize(img.getWidth()), DisplayUtils.getScaledSize(img.getHeight()), null);
+            g.drawImage(img, 0, 0, 

```

Comment:
Setting rendering hints seems to help with the scaling artifacts:
```Java
Map<RenderingHints.Key, Object> hints = new HashMap<RenderingHints.Key, Object>();
hints.put(RenderingHints.KEY_INTERPOLATION, RenderingHints.VALUE_INTERPOLATION_BICUBIC);
hints.put(RenderingHints.KEY_RENDERING, RenderingHints.VALUE_RENDER_QUALITY);
hints.put(RenderingHints.KEY_ANTIALIASING, RenderingHints.VALUE_ANTIALIAS_ON);
((Graphics2D) g).addRenderingHints(hints);
```
(From `org.fife.ui.rtextarea.BufferedImageBackgroundPainterStrategy`.)